src/tenants/common/Home/index.tsx

```diff
@@ -14,9 +14,14 @@ const { useTranslation } = i18next;
 interface Props {
   leaderboard: any;
   tenantScore: any;
+  updateTenantScore: string;
```

Comment:
I'm not clear how this is used. I don't see any place in the app that uses updateTenantScore